vector/src/main/java/im/vector/app/features/roomprofile/polls/RoomPollsFragment.kt

```diff
@@ -67,6 +67,7 @@ class RoomPollsFragment : VectorBaseFragment<FragmentRoomPollsBinding>() {
         tabLayoutMediator = TabLayoutMediator(views.roomPollsTabs, views.roomPollsViewPager) { tab, position ->
             when (position) {
                 0 -> tab.text = getString(R.string.room_polls_active)
+                1 -> tab.text = getString(R.string.room_polls_ended)
```

Comment:
maybe declare some constants for tab indexes to avoid confusions?

Reply:
Agree, updated in [85cfa43](https://github.com/vector-im/element-android/pull/7886/commits/85cfa433d9e15362c0b907d19e253c01cff665c5)